src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -60,6 +60,13 @@ public int MaximumFollowRelLink
             set { base._maximumFollowRelLink = value; }
         }
 
+        /// <summary>
+        /// Gets or sets the ResponseHeadersVariable property.
+        /// </summary>
+        [Parameter]
+        [Alias("HV")]
+        public string ResponseHeadersVariable { get; set; }
```

Comment:
Should we add any validation attributes?

Reply:
`SessionVariable` does something similar and it doesn't have any validation. I'm not sure what value it would add. Aside from being null or empty (which is tested to set it on line 99) a variable name can be any valid string.